### PR TITLE
[ui-storageBrowser] Add cuix and antd classes to Storage Browser

### PR DESCRIPTION
**Pull request by nidhibhatg**
## What changes were proposed in this pull request?

- Added cuix and antd classes to storage browser page
- Removed redundant styling from all storage browser components
- Removed React.FC from the components

## How was this patch tested?

- Manually

Please review [Hue Contributing Guide](https://github.com/cloudera/hue/blob/master/CONTRIBUTING.md) before opening a pull request.

### CODE REVIEW

File: desktop/core/src/desktop/js/apps/storageBrowser/StorageBrowserPage/StorageBrowserTabContents/StorageBrowserTabContent.tsx [hunk 1]
```diff
@@ -193,22 +194,45 @@ const StorageBrowserTabContent: React.FC<StorageBrowserTabContentProps> = ({
       });
   }, [filePath, pageSize, pageNumber, sortByColumn, sortOrder, refreshKey]);
 
+  const tempBreadcrumbs = [
```
Comment (bjornalm) [inline desktop/core/src/desktop/js/apps/storageBrowser/StorageBrowserPage/StorageBrowserTabContents/StorageBrowserTabContent.tsx:197]: What is this?

File: desktop/core/src/desktop/js/reactComponents/FileChooser/PathBrowser/PathBrowser.scss [hunk 2]
```diff
@@ -19,82 +19,77 @@
...
   }
-}
 
-.hue-path-browser__breadcrumb-seperator {
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  flex: 0 0 auto;
+  .hue-path-browser__input {
+    width: 300px;
```
Comment (bjornalm) [inline desktop/core/src/desktop/js/reactComponents/FileChooser/PathBrowser/PathBrowser.scss:79]: There are multiple hardcoded widths in this style, do they have some relation or are they completely independent of each other?

**Review by bjornalm — APPROVED** [review]
Nice work! Please see feedback and questions before merging.

File: desktop/core/src/desktop/js/apps/storageBrowser/StorageBrowserPage/StorageBrowserTabContents/StorageBrowserTabContent.tsx (hunk 1 above)
Comment (nidhibhatg) [reply desktop/core/src/desktop/js/apps/storageBrowser/StorageBrowserPage/StorageBrowserTabContents/StorageBrowserTabContent.tsx:197]: Sorry missed it. WIll remove it

File: desktop/core/src/desktop/js/apps/storageBrowser/StorageBrowserPage/StorageBrowserTable/StorageBrowserTable.scss [hunk 3]
```diff
@@ -19,57 +19,44 @@
...
-  }
-
-  thead {
-    //TODO:Check why height isnt applied in safari browser
-    height: $cell-height;
-    background-color: vars.$fluidx-gray-040;
+  .hue-storage-browser__table-row {
+    td.ant-table-cell {
+      height: $cell-height;
+      white-space: nowrap;
```
Comment (JohanAhlen) [inline desktop/core/src/desktop/js/apps/storageBrowser/StorageBrowserPage/StorageBrowserTable/StorageBrowserTable.scss:45]: Nit: mixin available in src/desktop/js/components/styles/_mixins.scss as an FYI

**Review by JohanAhlen — APPROVED** [review]
Nice!

File: desktop/core/src/desktop/js/reactComponents/FileChooser/PathBrowser/PathBrowser.scss (hunk 2 above)
Comment (nidhibhatg) [reply desktop/core/src/desktop/js/reactComponents/FileChooser/PathBrowser/PathBrowser.scss:80]: Independent

**Pull request merged**